README.md

```diff
@@ -1,19 +1,218 @@
+repositories like any other GitHub App. Follow the [quick start
+instructions](quick-start.md) to setup the configuration files needed to enable
+Allstar on your repositories.
 
+These quick start instructions don't provide much explanation, for more details
+on all the configuration options, continue reading.
 
-# 
-**Get Involved**
+### [Help! I'm getting issues created by Allstar and I don't want them.](opt-out.md)
 
-A part of the [Securing Critical Projects WG](https://github.com/ossf/wg-securing-critical-projects).
+## **Enable Configuration**
+
+Allstar can be enabled on individual repositories at the app level, and
+additionally each security policy can be enabled or disabled as well. For
+organization-level configuration, create a repository named `.allstar` in your
+organization. Then create a file called `allstar.yaml` in that
+repository.
+
+Allstar can either be set to an opt-in or opt-out strategy. In opt-in, only
+those repositories explicitly listed are enabled. In opt-out, all repositories
+are enabled, and repositories may be explicitly listed to opt-out. Allstar is
+set to opt-in by default, and therefore is not enabled on any repository
+immediately after installation. To continue with the default opt-in strategy,
+list the repositories for Allstar to be enabled on in your organization like so:
+
+```
+optConfig:
+  optInRepos:
+  - repo-one
+  - repo-two
+```
+
+To switch to the opt-out strategy, set that option to true:
+
+```
+optConfig:
+  optOutStrategy: true
+```
+
+If you with to enable Allstar on all but a few repositories, you may use opt-out
+and list the repositories to disable:
+
+```
+optConfig:
+  optOutStrategy: true
+  optOutRepos:
+  - repo-one
+  - repo-two
+```
+
+### Repository Override
+
+Individual repositories can also opt in or out using configuration files inside
+those repositories. For example, if the organization is configured with the
+opt-out strategy, a repository may opt itself out by including the file
+`.allstar/allstar.yaml` with the contents:
+
+```
+optConfig:
+  optOut: true
+```
+
+Conversely, this allows repositories to opt-in and enable Allstar when the
+organization is configured with the opt-in strategy. Because opt-in is the
+default strategy, this is how Allstar works if the `.allstar` repository doesn't
+exist.
+
+In the organization-level `allstar.yaml`, repository override may be disabled with the setting:
+
+```
+optConfig:
+  disableRepoOverride: true
+```
+
+This allows an organization-owner to have a central point of approval for
+repositories to request an opt-out through a GitHub PR. Understandably, Allstar
+or individual policies may not make sense for all repositories.
+
+### Policy Enable
+
+Each individual policy configuration file (see below) also contains the exact
+same `optConfig` configuration object. This allows granularity to enable
+policies on individual repositories. A policy will not take action unless both
+it is enabled and Allstar is enabled as a whole.
```

Comment:
```suggestion
policies on individual repositories. A policy will not take action unless
it is enabled **and** Allstar is enabled as a whole.
```
both it is... read weird, I'm not sure this suggestion is any better though

Reply:
both is weird
i like Appu's suggestion or split into two bullet points.